### ISSUE TITLE
Missing descriptive alt texts in most images

### ISSUE THREAD

**Issue by TheEvilSkeleton** [Vanilla-OS/website#63]

**Comment by kbdharun** [comment]
I agree Alt images are essential, will add them to the docs to-do list.

**Comment by orowith2os** [comment]
I'll help out with alt text as a whole, and I'd like to be included in everything regarding accessibility from now on.
